src/main/java/com/minecolonies/entity/ai/citizen/lumberjack/EntityAIWorkLumberjack.java

```diff
@@ -414,7 +414,8 @@ private void tryGettingUnstuckFromLeaves()
      */
     private boolean plantSapling(@NotNull BlockPos location)
     {
-        if (BlockPosUtil.getBlock(world, location) != Blocks.AIR)
+        Block worldBlock = world.getBlockState(location).getBlock();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'worldBlock' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)